.circleci/config.yml

```diff
@@ -536,6 +577,91 @@ jobs:
     steps:
       - checkout
 
+  importers:
+    parameters:
+      python_version_major:
+        type: integer
+        default: 3
+      python_version_minor:
+        type: integer
+        default: 8
+      shard:
+        type: string
+        default: "core"
+      toxenv:
+        type: string
+      codecov_flags:
+        type: string
+        default: ""
+      notify_on_failure:
+        type: boolean
+        default: false
+      xdist:
+        type: integer
+        default: 3
+      tox_args:
+        type: string
+        default: ""
+    machine: true
+    resource_class: xlarge
+    working_directory: ~/project
+    environment:
+      SHARD: << parameters.shard >>
+    steps:
+      - checkout
+      - run:
+          name: Install system deps
+          command: |
+            sudo apt-get update && sudo apt-get install -y libsndfile1 ffmpeg
+      - gcloud/install
+      - setup_gcloud
+      - run:
+          name: Decode GCP Service Account JSON
+          command: echo $GCP_SERVICE_ACCOUNT_JSON_ENCODED | base64 --decode > ${HOME}/gcp-key.json
+      - run:
+          name: Authenticate with GCP Artifact Registry
+          command: |
+            echo $GCP_SERVICE_ACCOUNT_JSON_DECODED | docker login -u _json_key --password-stdin https://<< pipeline.parameters.gcp_project >>-docker.pkg.dev
+      - start_local_testcontainer
+      - start_local_testcontainer:
+          local_base_port: "9180"
+          services_api_port: "9183"
+          fixture_service_port: "9115"
+          container_name: "wandb-local-testcontainer2"
+          volume_name: "wandb-local-testcontainer-vol2"
+      - run:
+          name: Install Python dependencies
+          command: |
+            pyenv install << parameters.python_version_major >>.<< parameters.python_version_minor >>
+            pyenv local << parameters.python_version_major >>.<< parameters.python_version_minor >>
+            pip install -U tox nox pip
+            while ! curl http://localhost:8080/healthz; do sleep 1; done
```

Comment:
This isn't installing a Python dependency, so it should be in a separate named step.

Also, should this wait for the second testcontainer to start up as well? And maybe it should have a timeout, or else it could hang CI and increase costs.

Reply:
Thanks, will add a wait for the second testcontainer.

This step is a based on a [similar step](https://github.com/wandb/wandb/blame/andrew/importers2/.circleci/config.yml#L727-L732) for the job `pytest`.  It has a timeout of `no_output_timeout: 5m`, which matches the pytest job.

Do you think we should change both of the steps for consistency?  And if so, what to?

Reply:
Ah, I see the timeout now. Yes, it would be good for the other job to do the wait in a separate step as well.

```yaml
- run:
    name: Install Python dependencies
    no_output_timeout: 5m
    ...
- run:
    # In this job, you'd wait for both test containers here
    name: Wait for test container startup
    no_output_timeout: 1m
    ...
```

This also lets you use a separate shorter timeout for the test container startup, which is a nice benefit.